plugin/src/main/cpp/extensions/openxr_fb_composition_layer_secure_content_extension_wrapper.cpp

```diff
@@ -92,7 +97,7 @@ uint64_t OpenXRFbCompositionLayerSecureContentExtensionWrapper::_set_viewport_co
 
 	switch ((ExternalOutput)(int)p_property_values.get(EXTERNAL_OUTPUT_PROPERTY_NAME, EXTERNAL_OUTPUT_DISPLAY)) {
```

Comment:
Can you reuse the `external_output` variable from above.

Reply:
Ah, yes, we can! Updated in my latest push